pandas/core/window.py

```diff
@@ -850,7 +850,41 @@ def mean(self, *args, **kwargs):
         return self._apply('roll_mean', 'mean', **kwargs)
 
     _shared_docs['median'] = dedent("""
-    %(name)s median
+    Calculate the %(name)s median.
+
+    Parameters
+    ----------
+    **kwargs
+        Under Review.
+
+    Returns
+    -------
+    Series or DataFrame
+        Returned object type is determined by the caller of the %(name)s
+        object.
+
+    See Also
+    --------
+    Series.%(name)s : Calling object with Series data
+    DataFrame.%(name)s : Calling object with DataFrames
+    Series.median : Equivalent method for Series
+    DataFrame.median : Equivalent method for DataFrame
+
+    Examples
+    --------
+    The below example will show a rolling calculation with a window size of
```

Comment:
I'd be more direct and say "Compute the rolling mean of a series with a window size of 3:".

Reply:
I agree... but this is median. Copy paste has failed me!

Reply:
Ah yes median, sorry.